### PR TITLE
RM-7672 Make re-motion jQuery private by using jQuery.noConflict

### PR DESCRIPTION
**Pull request by a-ctor**

### CODE REVIEW

File: Remotion/Web/Core/Res/HTML/jquery.iFrameShim.js [hunk 1]
```diff
@@ -69,4 +69,4 @@
         });
       }
       : function (s) { return this; };
-})(jQuery);
\ No newline at end of file
+})(Remotion.jQuery);
```
Comment (MichaelKetting) [inline Remotion/Web/Core/Res/HTML/jquery.iFrameShim.js:72]: we need to make `HtmlHeadAppenderExtensions.RegisterJQueryIFrameShimJavaScriptInclude` inline with with  `HtmlHeadAppenderExtensions.RegisterUtilitiesJavaScriptInclude` while leaving the method as obsolete-as-error with a notsupportedexception in the body. please check other usages of ObsoleteAttribute for examples. => breaking change

**Review by MichaelKetting — CHANGES_REQUESTED** [review]

File: Remotion/Web/Core/UI/HtmlHeadAppenderExtensions.cs [hunk 2]
```diff
@@ -46,13 +46,12 @@ public static void RegisterUtilitiesJavaScriptInclude (this HtmlHeadAppender htm
     /// <summary>
     /// Registers jquery.IFrameShim.js in non-themed HTML folder of Remotion.Web.dll.
     /// </summary>
+    [Obsolete ("JQuery iFrame shim was only needed for IE, which is no longer supported. (Version 3.0.0)", true)]
```
Comment (MichaelKetting) [inline Remotion/Web/Core/UI/HtmlHeadAppenderExtensions.cs:49]:
```suggestion
    [Obsolete ("JQuery iFrame shim was only needed for IE, which is no longer supported. (Version 3.0.0-alpha.12)", true)]
```

File: Remotion/Web/Core/UI/HtmlHeadAppenderExtensions.cs [hunk 3]
```diff
@@ -46,13 +46,12 @@ public static void RegisterUtilitiesJavaScriptInclude (this HtmlHeadAppender htm
...
     public static void RegisterJQueryIFrameShimJavaScriptInclude (this HtmlHeadAppender htmlHeadAppender)
     {
       ArgumentUtility.CheckNotNull ("htmlHeadAppender", htmlHeadAppender);
 
-      string key = typeof (HtmlHeadContents).FullName + "_JQueryBgiFrames";
-      var href = ResourceUrlFactory.CreateResourceUrl (typeof (HtmlHeadContents), ResourceType.Html, "jquery.IFrameShim.js");
-      htmlHeadAppender.RegisterJavaScriptInclude (key, href);
+      throw new NotSupportedException ("JQuery iFrame shim was only needed for IE, which is no longer supported. (Version 3.0.0)");
```
Comment (MichaelKetting) [inline Remotion/Web/Core/UI/HtmlHeadAppenderExtensions.cs:54]:
```suggestion
      throw new NotSupportedException ("JQuery iFrame shim was only needed for IE, which is no longer supported. (Version 3.0.0-alpha.12)");
```

File: Remotion/Web/Core/UI/Controls/DropDownMenuImplementation/Rendering/DropDownMenuRenderer.cs [hunk 4]
```diff
@@ -71,7 +71,10 @@ public void RegisterHtmlHeadContents (HtmlHeadAppender htmlHeadAppender)
       ArgumentUtility.CheckNotNull ("htmlHeadAppender", htmlHeadAppender);
 
       htmlHeadAppender.RegisterUtilitiesJavaScriptInclude();
-      htmlHeadAppender.RegisterJQueryIFrameShimJavaScriptInclude();
+
+      string jQueryIframeShimScriptKey = typeof (HtmlHeadContents).FullName + "_JQueryBgiFrames";
+      var href = ResourceUrlFactory.CreateResourceUrl (typeof (HtmlHeadContents), ResourceType.Html, "jquery.IFrameShim.js");
+      htmlHeadAppender.RegisterJavaScriptInclude (jQueryIframeShimScriptKey, href);
```
Comment (MichaelKetting) [inline Remotion/Web/Core/UI/Controls/DropDownMenuImplementation/Rendering/DropDownMenuRenderer.cs:77]: Something went wrong in the review comment. I meant we should inline the relevant code into the `RegisterUtilitiesJavaScriptInclude` method. It's going to get dropped anyway when we delte the IE code later this release.

**Review by MichaelKetting — CHANGES_REQUESTED** [review]
Sorry, communications hickup

File: Remotion/Web/Core/UI/Controls/DropDownMenuImplementation/Rendering/DropDownMenuRenderer.cs (hunk 4 above)
Comment (a-ctor) [reply Remotion/Web/Core/UI/Controls/DropDownMenuImplementation/Rendering/DropDownMenuRenderer.cs:77]: What relevant code? I still don't follow

Comment (MichaelKetting) [reply Remotion/Web/Core/UI/Controls/DropDownMenuImplementation/Rendering/DropDownMenuRenderer.cs:77]: These three lines of code. I didn't mean to inline them 3 times in various files, just once into the `RegisterUtilitiesJavaScriptInclude` method.

**Review by MichaelKetting — APPROVED** [review]

**Pull request merged**